docs/source/intro-clx-streamz.ipynb

```diff
@@ -39,140 +39,167 @@
-    "```\n",
-    "\n",
     "Processed data will be pushed to the kafka topic named `output`. To view all processed output run:\n",
-    "```\n",
+    "\n",
+    "`\n",
     "docker exec cybert-streamz bash -c 'source activate rapids && $KAFKA_HOME/bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic output --from-beginning'\n",
-    "```"
+    "`"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Capturing Benchmarks\n",
+    "Your Quickstart Docker container includes the data and model required to run both cyBERT and DGA detection stream processing.\n",
+    "\n",
+    "## Run cyBERT Streamz Example on Apache Logs\n",
     "\n",
-    "To capture benchmarks add the benchmark flag along with average log size (kb), for throughput (mb/s) and average batch size (mb) estimates, to the docker run command above\n",
     "```\n",
-    "docker run -it \n",
-    "--gpus '\"device=0,1,2\"' \\\n",
-    "-p 8787:8787 \\\n",
-    "--name cybert-streamz -d cybert-streamz:latest \\\n",
-    "--broker localhost:9092 \\\n",
-    "--group_id streamz \\\n",
-    "--input_topic input \\\n",
-    "--output_topic output \\\n",
-    "--model_file /path/to/model.pth \\\n",
-    "--label_file /path/to/label.json \\\n",
-    "--poll_interval 1s \\\n",
-    "--max_batch_size 1000 \\\n",
-    "--data /path/to/dataset \\\n",
-    "--benchmark 20\n",
+    "bash $CLX_STREAMZ_HOME/scripts/kafka_topic_setup.sh \\\n",
+    "    -i cybert_input \\\n",
+    "    -o cybert_output \\\n",
+    "    -d $CLX_STREAMZ_HOME/data/apache_raw_sample_1k.txt\n",
     "```\n",
     "\n",
-    "To print benchmarks to the docker logs send a SIGINT signal to the running cybert process\n",
     "```\n",
-    "# To get the PID\n",
-    "$ docker exec cybert-streamz ps aux | grep \"cybert\\.py\" | awk '{print $2}'\n",
-    "# Kill process\n",
-    "$ docker exec cybert-streamz kill -SIGINT <pid>\n",
-    "$ docker logs cybert-streamz\n",
-    "```"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Run cyBERT Streamz Example on Apache Logs\n",
+    "python $CLX_STREAMZ_HOME/python/cybert.py \\\n",
+    "    --broker localhost:9092 \\\n",
+    "    --input_topic cybert_input \\\n",
+    "    --output_topic cybert_output \\\n",
+    "    --group_id streamz \\\n",
+    "    --model $CLX_STREAMZ_HOME/ml/models/cybert_pytorch_model.bin \\\n",
+    "    --label_map $CLX_STREAMZ_HOME/ml/labels/cybert_config.json \\\n",
+    "    --poll_interval 1s \\\n",
+    "    --max_batch_size 500 \\\n",
+    "    --dask_scheduler localhost:8786 \\\n",
+    "    --benchmark 10\n",
+    "```\n",
     "\n",
-    "Your Quickstart Docker container includes the data and model required to run cyBERT stream processing on a sample of Apache logs.\n",
+    "## Run DGA Streamz Example on Sample Domains\n",
+    "\n",
+    "```\n",
+    "bash $CLX_STREAMZ_HOME/scripts/kafka_topic_setup.sh \\\n",
```

Comment:
As it is now it seems you have to run two commands to run this dga example (same with cybert). Can we minimize this to just one "quick start" command? By either maybe creating a wrapper script, or creating and preloading the kafka topics with the sample data within the docker entry script. Open to ideas

Reply:
Also might help to preclude this command with `docker exec clx-streamz bash -c <command>` as needed. 

Can we also move this portion of the README closer to the top? To highlight an example of how to get started with dga/cybert quickly

Reply:
sure, will create a wrapper script to trigger pipelines. But let's discuss in the sprint planning meeting to get @efajardo-nv opinion as well.

Reply:
@brhodes10 updated docs as per the suggestion.